## INCLUDE THIS IN task_supervisor.yaml ##

  - {name: docking_handler, type: 7, class: 'docking_handler::DockingHandler'}

docking_handler:
  watchdog_rate: 2.0
  watchdog_timeout: 0
  docking_launch_package: "pallet_launcher"
  docking_launch_file: "pallet_main_launcher.launch"
  docking_launch_nodes: "/camera_lidar_docking /camera_docking /navigation_based_docking"
